lib/src/executables/mui_migration.dart

```diff
@@ -20,21 +20,60 @@ import 'package:glob/glob.dart';
 import 'package:glob/list_local_fs.dart';
 import 'package:logging/logging.dart';
 import 'package:over_react_codemod/src/ignoreable.dart';
-import 'package:over_react_codemod/src/mui_suggestors/mui_button_group_migrator.dart';
-import 'package:over_react_codemod/src/mui_suggestors/mui_button_migrator.dart';
-import 'package:over_react_codemod/src/mui_suggestors/mui_button_toolbar_migrator.dart';
 import 'package:over_react_codemod/src/mui_suggestors/mui_importer.dart';
+import 'package:over_react_codemod/src/mui_suggestors/mui_migrators.dart';
 import 'package:over_react_codemod/src/util.dart';
 import 'package:over_react_codemod/src/util/package_util.dart';
 import 'package:over_react_codemod/src/util/pubspec_upgrader.dart';
 
 final _log = Logger('orcm.mui_migration');
+const _componentFlag = 'component';
 
 void main(List<String> args) async {
-  final parser = ArgParser.allowAnything();
+  final parser = ArgParser()
+    ..addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Prints this help output.',
+    )
+    ..addFlag(
```

Comment:
I think `ArgParser.allowAnything` plus `runInteractiveCodemod`'s `additionalHelpOutput` makes it so that we don't have to redeclare all of these; was there a reason you switched from `allowAnything`?

Not a huge deal, just curious.

Reply:
The reason was because the `AllowAnythingParser` doesn't support the `addMultiOption` method (or any of those similar methods). I couldn't figure out a way to add an option without having to redeclare all the default options, but maybe there is a way I don't know of?

Reply:
The reason was because the `AllowAnythingParser` [doesn't support](https://github.com/dart-lang/args/blob/a85ab5e2d10640cb546bdff2866abcea1920b62c/lib/src/allow_anything_parser.dart#L75) the `addMultiOption` method (or any of those similar methods). I couldn't figure out a way to add an option without having to redeclare all the default options, but maybe there is a way I don't know of?